src/main/java/com/xebia/headerbuddy/SwaggerConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.xebia.headerbuddy;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.Collections;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+
+    @Bean
+    public Docket api() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .select()
+                .apis(RequestHandlerSelectors.basePackage("com.xebia.headerbuddy.controllers"))
```

Comment:
Easier to use:                 `.apis(RequestHandlerSelectors.basePackage(HeaderBuddyController.class.getPackage().getName()))`

more "refactor safe"